### DISCUSSION TITLE
Adding an alternative to telescope for /file command. specifically mini.pick from mini.nvim

### DISCUSSION THREAD

**Discussion by bassamsdata** [olimorris/codecompanion.nvim#166] [Ideas]
Hello,

Since `#buffers` is no longer available, which I used daily and really liked, I ended up rolling back to the previous version because I don’t use Telescope, and it's the only way to use the new slash `/file` command. For the slash `/buffers`, I was able to use dressing.nvim.

If you’re okay with it, I created a PR to add mini.pick.

thank you

**Comment by olimorris** [comment]
I'll explain why I removed `#buffers` in the announcement. But it wasn't very practical because it sent _all_ buffers that were loaded and listed. So it was difficult to visually know what had been sent to the LLM. the slash command improves on this significantly.

Thanks so much for the PR. This is exactly what I was hoping the community would develop.
